### PR TITLE
Fix 494

### LINKED ISSUE #494: The citizens text under pictogram setting also effect the guardian
**Describe the bug**
When changing if a citizen should be able to see text on the activities it also changes the setting for guardians. 

**To Reproduce**
1. Go to the settings for a citizen
2. Click on remove the checkmark from Piktogram tekst er synlig
3. Go to a weekplan for that citizen in guardian mode

**Expected behavior**
There should still be text under the activities for the guardians.

**Actual behavior**
There is no text under the activities for the guardians. 

**Screenshots**
image

**Environment (please complete the following information):**
- OS: Android
- Emulator Pixel 7
- APK Version 29

### PR DESCRIPTION
**Pull request by klogeat**
Pictogram text is now always displayed when in guardian mode

### CODE REVIEW

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw610f20 you have been assigned this. Here's your checklist.
Please use the following checklist. Feel free to check it off as you go along.

**Code Design**

- [ ] The code is in the right place? (Both in terms of folder structure and class structure)
- [ ] This code could not, to the best of my knowledge, have reused existing code
- [ ] The code is not over-engineered. Examples of over-engineering: Implemented behavior for future problems
- [ ] The code does it introduce functionality that is not necessary for solving the problem

**Code Readability and Maintainability**

- [ ] Names are meaningful and self-documenting
- [ ] It is understandable, by reading the code, what it does
- [ ] The code is simple and readable. i.e. it contains no "hack", or obscure solution
- [ ] All new functionality is covered by tests
- [ ] It is understandable, by reading the tests, what they do
- [ ] The tests covers sensible cases. Both happy paths and exception paths
- [ ] The tests cover the full functionality. i.e., The tests fails if some of the requested functionality is missing
- [ ] Error messages in the code are understandable
- [ ] Confusing sections of code are covered by comments

**Functionality**

For this part, open the app, and test:

- [ ] The features that the code claims to implement, are actually implemented
- [ ] The code does not look like it has bugs
- [ ] All the new screens are reachable through in-app navigation

**Points for Discussion and Reflection**

- [ ] UI messages, that the user might meet, are actually helpful
- [ ] There are no errors that are bad practice. E.g. hard-coding a test database connection
- [ ] There is well written code in this. Remember to tell the developer!

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw614f20 you have been assigned this. Here's your checklist.
Please use the following checklist. Feel free to check it off as you go along.

**Code Design**

- [ ] The code is in the right place? (Both in terms of folder structure and class structure)
- [ ] This code could not, to the best of my knowledge, have reused existing code
- [ ] The code is not over-engineered. Examples of over-engineering: Implemented behavior for future problems
- [ ] The code does it introduce functionality that is not necessary for solving the problem

**Code Readability and Maintainability**

- [ ] Names are meaningful and self-documenting
- [ ] It is understandable, by reading the code, what it does
- [ ] The code is simple and readable. i.e. it contains no "hack", or obscure solution
- [ ] All new functionality is covered by tests
- [ ] It is understandable, by reading the tests, what they do
- [ ] The tests covers sensible cases. Both happy paths and exception paths
- [ ] The tests cover the full functionality. i.e., The tests fails if some of the requested functionality is missing
- [ ] Error messages in the code are understandable
- [ ] Confusing sections of code are covered by comments

**Functionality**

For this part, open the app, and test:

- [ ] The features that the code claims to implement, are actually implemented
- [ ] The code does not look like it has bugs
- [ ] All the new screens are reachable through in-app navigation

**Points for Discussion and Reflection**

- [ ] UI messages, that the user might meet, are actually helpful
- [ ] There are no errors that are bad practice. E.g. hard-coding a test database connection
- [ ] There is well written code in this. Remember to tell the developer!

File: lib/widgets/pictogram_text.dart [hunk 1]
```diff
@@ -23,40 +25,59 @@ class PictogramText extends StatelessWidget {
...
+    if (weekModeSnapshot.data == WeekplanMode.guardian) {
+      return true;
+    }
+    return false;
+  }
+
+  SizedBox _buildPictogramText(BuildContext context, String pictogramText) {
+    return SizedBox(
+        width: MediaQuery.of(context).size.width,
+        height: MediaQuery.of(context).size.width / 4,
```
Comment (frederikspang) [inline lib/widgets/pictogram_text.dart:65]: Er det meningen, at vi kører med en højde, der er relativ til bredden? I så fald, fint!

File: lib/widgets/pictogram_text.dart [hunk 2]
```diff
@@ -23,40 +25,59 @@ class PictogramText extends StatelessWidget {
...
+                return Container(width: 0, height: 0);
+              });
         });
   }
+
+  bool _isGuardianMode(AsyncSnapshot<WeekplanMode> weekModeSnapshot) {
+    if (weekModeSnapshot.data == WeekplanMode.guardian) {
+      return true;
+    }
+    return false;
```
Comment (frederikspang) [inline lib/widgets/pictogram_text.dart]:
```suggestion
     return weekModeSnapshot.data == WeekplanMode.guardian;
```

**Review by frederikspang — DISMISSED** [review]

**Review by frederikspang — APPROVED** [review]

**Review by Simon97 — APPROVED** [review]

**Pull request merged**